### ISSUE TITLE
[@portabletext/vue] Unknown block type "block", specify a component for it in the `components.types` prop

### ISSUE THREAD

**Issue by mikehwagz** [portabletext/vue-portabletext#39]
I have a few link annotations in a field and attempting to render on the page but seeing this strange warning. Typically I'd expect this for unknown types, not `block`. Any idea what is going on here?

**Comment by stipsan** [comment]
@rdunk sounds like stega snuck in here 🤔

**Comment by rdunk** [comment]
As in stega got into the block type str?

This one is going over my head, any more details?  😆

**Comment by stipsan** [comment]
>As in stega got into the block type str?
@rdunk 

Yes, it should be skipped by default. But custom filter options could re-enable it. That's the scenario where I've seen this error myself. We might want to update `@portabletext` libs to always strip out stega from areas like `type` or anything else that would break render logic in similar ways as to here
